.sass-lint.yml

```diff
@@ -0,0 +1,59 @@
+#########################
+## Sample Sass Lint File
+#########################
+# Linter Options
+options:
+  # Don't merge default rules
+  merge-default-rules: false
+  # Set the formatter to 'html'
+  formatter: html
+  # Output file instead of logging results
+  # output-file: 'linters/sass-lint.html'
+  # Raise an error if more than 50 warnings are generated
+  max-warnings: 50
+# File Options
+files:
+  include: '**/*.s+(a|c)ss'
+  ignore:
+    - 'sass/vendor/**/*.*'
+    - '.gems/**/*.*'
+    - 'node_modules/**/*.*'
+    - 'coverage/**/*.*'
+    - '**/_toast.sass'
```

Comment:
Ignores toast because sass-lint doesn't allow this:
```#toast-container```
It should be a class instead of an ID, but it seemed to me this config is inherent to the gem.